packages/fxa-payments-server/scripts/test-ci.sh

```diff
@@ -1,25 +1,10 @@
 #!/bin/bash -ex
 
-DIR=$(dirname "$0")
-
-cd "$DIR/../../../"
-
-npx lerna bootstrap \
-  --scope fxa-shared \
-  --scope fxa-geodb \
-  --scope fxa-settings \
-  --scope fxa-content-server \
-  --scope fxa-payments-server \
-  --concurrency 2
-
-cd packages/fxa-payments-server
-
-cd ../fxa-react
-npm ci
-
-cd ../fxa-payments-server
-npm ci
-
-# TODO rm the CI=false
-PUBLIC_URL=/ INLINE_RUNTIME_CHUNK=false CI=false npm run build
-CI=yes npm test
+# yarn workspaces focus fxa-payments-server
+
+# payments-server has some missing dependency that
+# causes tests to fail when only it's deps are installed
+# but passes when all fxa deps are installed
```

Comment:
i have no idea what's causing this, but here's what it looks like: https://app.circleci.com/pipelines/github/mozilla/fxa/10575/workflows/8d14f766-9196-4c58-9715-5c11b90e40de/jobs/168829

Reply:
Wondering if this is related to the issues pointed out here:
https://medium.com/@rossbulat/typescript-working-with-paths-packages-and-yarn-workspaces-6fbc7087b325

Also, I assume that when we want to use fxa-shared in a typescript package, we will need the path alias and tsconfig changes the above post cites?

Reply:
There's still work to do after this PR to improve our typescript situation (thanks for the link) but I think this is a missing dependency, I just don't know which.

Reply:
I think payments-server relies on some assets built by content-server, not sure if that's the cause here.

Reply:
which assets? I didn't see any references to content-server outside of scss files. FWIW this also fails in the same way; builds fine, tests fail:
```
yarn workspaces focus fxa-payments-server
yarn workspace fxa-content-server run build
yarn build
yarn test
```